integration/kubernetes/lib.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
It just contains a function that is used in the scripts and test, it may grow up as we add more tests.
I added a comment of the purpose inside the file